tests/PHPStan/Analyser/data/array-intersect-key.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace ArrayIntersectKey;
+
+use function array_intersect_key;
+use function PHPStan\Testing\assertType;
+
+class Foo
```

Comment:
Hi, file renames are hard to review, there isn't a good reason to do it :) So please keep the original file name. You can follow my advice from the other PR, and keep testing it on PHP 7 :)

Reply:
adapted. I just moved up the yield of the constant test file so that they are all together and don't feel lonely :)